FFTConfig.scala

```diff
@@ -0,0 +1,107 @@
+package fftgenerator
+
+import breeze.math.{Complex}
```

Comment:
Can you try to clean up some of these imports? I feel like some of them shouldn't work... like `craft`? In any case, being more explicit with what is being imported would be nice (like `import dsptools.numbers.{DspComplex, Real}`)